smart-contracts/contracts/BridgeBank/BridgeToken.sol

```diff
@@ -8,25 +8,44 @@ import "@openzeppelin/contracts/access/Ownable.sol";
  * @title BridgeToken
  * @dev Mintable, ERC20Burnable, ERC20 compatible BankToken for use by BridgeBank
  **/
-
 contract BridgeToken is ERC20Burnable, Ownable {
 
     uint8 private _decimals;
+    string public cosmosDenom;
 
-    constructor(string memory _name, string memory _symbol, uint8 _tokenDecimals)
-        public
+    constructor(string memory _name, string memory _symbol, uint8 _tokenDecimals, string memory _cosmosDenom)
         ERC20(_name, _symbol)
         Ownable()
     {
        _decimals = _tokenDecimals;
+       cosmosDenom = _cosmosDenom;
     }
 
+    /**
+     * @notice If sender is the owner, mints `amount` to `user`
+     * @param user Address of the recipient
+     * @param amount How much should be minted
+     * @return true if the operation succeeds
+     */
     function mint(address user, uint256 amount) public onlyOwner returns (bool) {
         _mint(user, amount);
         return true;
     }
 
+    /**
+     * @notice Number of decimals this token has
+     */
     function decimals() public override view returns (uint8) {
         return _decimals;
     }
+
+    /**
+     * @notice Sets the cosmosDenom
+     * @param denom The new cosmos denom
+     * @return true if the operation succeeds
+     */
+    function setDenom(string calldata denom) external onlyOwner returns (bool) {
```

Comment:
Looks like you need to rebase off the latest peggy2.0 branch to resolve this and make this contract use RBAC.

Reply:
Actually, the new token was IbcToken.sol instead of BridgeToken.sol.
I'll bring the changes from IbcToken.sol to BridgeToken.sol and adjust the tests.